src/bundler/bundle_v2.zig

```diff
@@ -1846,7 +1922,7 @@ pub const BundleV2 = struct {
                 this.graph.heap.gc(true);
             }
         }
-        var log = &resolve.completion.?.log;
+        const log = this.bundler.log;
```

Comment:
Ditto. We shouldn't change this unless we're confident there's no thread-safety issue.